docs/modules/ROOT/pages/faq.adoc

```diff
@@ -135,3 +135,19 @@ embassy-time = { git = "https://github.com/embassy-rs/embassy.git", rev = "e5fdd
 ----
 
 Note that the git revision should match any other embassy patches or git dependencies that you are using!
+
+== How can I optimize the speed of my embassy-stm32 program?
+
+* Make sure RCC is set up to go as fast as possible
+* Make sure link:https://docs.rs/cortex-m/latest/cortex_m/peripheral/struct.SCB.html[flash cache] is enabled
+* build with `--release`
+* Set the following keys for the release profile in your `Cargo.toml``:
+    ** `opt-level=s`
+    ** `lto=fat`
+    ** `build-std=core`
```

Comment:
Are these stable in `Cargo.toml` now? Or do they still need to be in `.cargo/config.toml`?

Reply:
Good question, I'm using nightly and it works from Cargo.toml. Though I got one notification in rustrover that `build-std=core` is unused. However, when I added it, it seemed like it was building std. I can look into it tomorrow.

Reply:
Thank you, very good catch @jamesmunns ! Looks like it is still unstable and has to go to `.cargo/config.toml`, see https://doc.rust-lang.org/cargo/reference/unstable.html

I will adapt the faq entry accordingly